web/testing/promise-api/index.html

```diff
@@ -53,12 +81,26 @@
       }).then(function() {
         loadKeyboards();
       });
+
+      function onLoad() {
```

Comment:
Can we go with `window.addEventListener('load')` instead of `body.onload`? And then this also probably belongs in eventLogging.js yes?

Reply:
I decided against that in part for the review process; some of the "removed code" was moved into this function.  As it's harder to track changes across files, and also because that code was part of the page's UI management, I left it.

Though, now that you've reviewed it and are comfortable with the changes, it'll be less of a jump to move it now if preferred.